boefjes/boefjes/plugins/kat_nmap_udp/main.py

```diff
@@ -43,17 +33,15 @@ def build_nmap_arguments(host: str, protocol: Protocol, top_ports: Optional[int]
 
 def run(boefje_meta: BoefjeMeta) -> List[Tuple[set, Union[bytes, str]]]:
     """Build Nmap arguments and return results to normalizer."""
-    top_ports = int(getenv("TOP_PORTS_UDP", TOP_PORTS_DEFAULT))
-    assert (
-        TOP_PORTS_MIN <= top_ports <= TOP_PORTS_MAX
-    ), f'{TOP_PORTS_MIN} <= {top_ports} <= {TOP_PORTS_MAX} fails. Check "TOP_PORTS" argument.'
+    top_ports = int(getenv("TOP_PORTS", TOP_PORTS_DEFAULT))
```

Comment:
Would it be wise to not collide this ENV var with the other TOP_PORTS var used in the TCP boefje? For now these vars are inserted via the schema, but maybe there's a future usecase for setting these vars directly on the boefje workers instead of via KAT's katalogus settings?

Reply:
Yes, This would clash with the feature used in boefjes/boefjes/job_handler.py where we ingest env vars prefixed with BOEFJE_ if no katalogus setting is present.

```
            if key.startswith("BOEFJE_"):
                katalogus_key = key.split("BOEFJE_", 1)[1]
                # Only pass the environment variable if it is not explicitly set through the katalogus,
                # if and only if they are defined in boefje.json
```

Reply:
I've renamed the UDP var back to TOP_PORTS_UDP.